docs/cloud/tcld/account/metrics/accepted-client-ca/remove.md

```diff
@@ -0,0 +1,93 @@
+---
+id: remove
+title: tcld account metrics accepted-client-ca remove
+sidebar_label: remove
+description: How to remove client CA certificates from the metrics endpoint of a Temporal Cloud account using tcld.
+tags:
+  - reference
+  - tcld
+---
+
+The `tcld account metrics accepted-client-ca remove` command removes client CA certificates from the metrics endpoint of a Temporal Cloud account.
+
+`tcld account metrics accepted-client-ca remove --ca-certificate <value>`
+
+Alias: `r`
+
+The following modifiers control the behavior of the command.
+
+### `--request-id`
+
+Specify a request identifier to use for the asynchronous operation. If not specified, the server assigns a request identifier.
+
+Alias: `-r`
+
+**Example**
+
+```bash
+tcld account metrics accepted-client-ca add --request-id <request_id> --ca-certificate <encoded_certificate>
+```
+
+### `--resource-version`
+
+Specify a resource version (ETag) to update from. If not specified, the latest version is used.
+
+Alias: `-v`
+
+**Example**
+
+```bash
+tcld account metrics accepted-client-ca add --resource-version <etag> --ca-certificate <encoded_certificate>
+```
+
+### `--ca-certificate`
+
+_Required modifier unless `--ca-certificate-fingerprint` or `--ca-certificate-file` is specified_
+
+Specify a base64-encoded string of a CA certificate PEM file.
+
+If `--ca-certificate-fingerprint` is also specified, both `--ca-certificate` and `--ca-certificate-file` are ignored.
+
+If `--ca-certificate-file` is also specified but `--ca-certificate-fingerprint` is not, only `--ca-certificate` is used.
+
+Alias: `-c`
+
+**Example**
+
+```bash
+tcld account metrics accepted-client-ca add --ca-certificate <encoded_certificate>
```

Comment:
this should be remove? Same with the other examples in this file. Not sure if this same copy-paste error exists for namespace ca tcld commands as well?

Reply:
🤦🏼‍♂️ Copy 'nd paste FTL...